packages/common/src/interfaces.ts

```diff
@@ -360,21 +360,6 @@ export interface IItemTemplate {
   estimatedDeploymentCostFactor: number;
 }
 
-export interface ICreateItemFromTemplateResponse {
```

Comment:
@MikeTschudi should this have been added back somewhere?

Reply:
oh wait...I see...when I look back at the source...it was a second instance of this